tools/development/start.sh

```diff
@@ -58,7 +58,7 @@ if [[ ! -z ${1} ]] && [[ ${1} == "--link" ]]; then
             cp -as /mnt/open-space-toolkit-core/include/OpenSpaceToolkit/Core /usr/local/include/OpenSpaceToolkit/Core; \
             cp -as /mnt/open-space-toolkit-core/src/OpenSpaceToolkit/Core/* /usr/local/include/OpenSpaceToolkit/Core/; \
             ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so /usr/local/lib/; \
-            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so.0 /usr/local/lib/;"
+            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so.1 /usr/local/lib/;"
```

Comment:
So these are the shared object "soname" of the dependencies of this project. I don't think we need to touch them right now. That being said we need to make this more generic across the repo (probably using the `.env` approach to have dependencies versions available in the Makefile)

Reply:
Reverted for now to so.0 and will propose an additional PR with `versions.env` file